docs/en/rules/Azure.Deployment.SecureValue.md

```diff
@@ -0,0 +1,94 @@
+---
+reviewed: 2022-10-10
+severity: Critical
+pillar: Security
+category: Infrastructure provisioning
+resource: Deployment
```

Comment:
The rule should apply to all resources in the template, not only to the ones under one of type Microsoft.Resources/deployments, why did you make that restriction?

Reply:
@verabe This has no bearing on the rule execution just documentation grouping. But we are trying to say this affects all deployments without being specific to template or bicep.

Deployments by there nature are mostly unexpanded. 

I am open to thoughts if it is more or less confusing then all resources. Technically they are both correct. All resources may be less confusing to general consumers.